seqr/utils/communication_utils.py

```diff
@@ -41,3 +43,28 @@ def send_welcome_email(user, referrer):
         password_token=user.password,
     )
     user.email_user('Set up your seqr account', email_content, fail_silently=False)
+
+
+def send_load_data_email(user, guid, namespace, name, individual_ids):
+    for data_manager in User.objects.filter(is_staff = True):
```

Comment:
Do not loop through all the users and send a different email to each one, you should send one email to all the users who need it. Instead of passing `[data_manager.email]` as the recipients, do something like `recipients = [user.email for user in User.objects.filter(is_staff = True)`.  This is an internal email so you can just take out the individualized greeting.